gemrb/core/Scriptable/Actor.cpp

```diff
@@ -2453,6 +2453,11 @@ Actor::stat_t Actor::ClampStat(unsigned int StatIndex, stat_t Value) const
 		return Value;
 	}
 
+	// don't clamp, it's a bit field
+	if (StatIndex == IE_SANCTUARY) {
+		return Value;
+	}
+
```

Comment:
Wait, why is this a problem? Is just the `maximum_values` entry of the stat set too low?